src/views/components/create-item-form.jsx

```diff
@@ -38,10 +38,12 @@ export default class CreateItemForm extends ListenerComponent {
             <div className='create-item-form'>
                 <h1>Create an Item</h1>
                 <form onSubmit={this.submit.bind(this)}>
-                    <select onChange={this.changeModel.bind(this)} >
-                        <option selected key='foo' value='bar' disabled>Choose a Model</option>
+                    <select defaultValue='default' onChange={this.changeModel.bind(this)} >
```

Comment:
This was a month ago, so I don't remember why I was told to do this, but this is what @mattmahn and @thofmann were telling me in slack, I believe.